script.js

```diff
@@ -1 +1 @@
-console.log('From javascript ')
+alert("This is a page created for the Reign fans! Enjoy!");
```

Comment:
all the files must have in the end a new empty line, add please a new empty line

Reply:
What?

Reply:
Why?

Reply:
![image](https://user-images.githubusercontent.com/25545942/61034502-14427180-a3ce-11e9-9b5e-66ed0e54761e.png)
github is complaining